tests/functiondef_docstring.py.uast

```diff
@@ -1,5 +1,6 @@
-Status:  ok
+Status:  error
 Errors: 
+ .  column out of bounds: 0 [1, 8]
```

Comment:
fuuuuuuuuuuuuuuuuu....!

Yes, found another instance of the same problem for other node types, I will try with a more generic solution with the fix for #58 so I don't have two release two versions more of pydetector.